### ISSUE TITLE
Parboiled2 - MergeNestedIfs "False Positive"

### ISSUE THREAD

**Issue by thomassuckow** [HairyFotr/linter#61]
Use of the parboiled2 library which uses scala macros causes a MergeNestedIfs warnings at every rule declaration

```
Parser.scala:6: warning: [MergeNestedIfs] These two nested ifs can be merged into one.
def Input = rule {
                 ^
```
